package.json

```diff
@@ -10,7 +10,7 @@
     "build": "npm run pre-build && npm run build:functions && svelte-kit build && npm run compile:sass",
     "build:functions": "cd functions && npm install && npm run build && cd ..",
     "build:web:emulator": "npm run pre-build && npm run config:web:demo && svelte-kit build -- --config-emulator-mode",
-    "watch:functions": "watch 'npm run build:functions && sleep 1 && npm run load:data' ./functions/src > /dev/null",
+    "watch:functions": "watch 'npm run build:functions && sleep 1 && npm run load:data' ./functions/src --ignoreDirectoryPattern='/generated/g' > /dev/null",
```

Comment:
Why ignore these, should we not reload if the generated glean files change?

Reply:
@rhelmer since `npm run build:functions` itself builds/re-generates the Glean files, it creates an infinite loop. I figured that changing the Glean definitions is such a rare/momentous occurrence that it's unlikely to happen very often, especially in the middle of a `npm run dev`, and also would almost definitely be accompanied by other code changes.

That said, there's definitely a way to do this that works without ignoring `/src/generated`, by splitting up `tsc` and the Glean build step (and then only run `tsc` in `watch:functions`). Let me know if you want me to make that change.

Reply:
Oh OK, we can do that separately thanks for the explanation.

Reply:
https://github.com/mozilla-rally/rally-web-platform/issues/357